dask/dataframe/io.py

```diff
@@ -798,9 +798,12 @@ def from_delayed(dfs, meta=None, divisions=None, prefix='from-delayed',
         An iterable of ``dask.delayed.Delayed`` objects, such as come from
         ``dask.delayed`` These comprise the individual partitions of the
         resulting dataframe.
-    $META
```

Comment:
OK, I should have checked `git blame`.  I'll replace it.  

Thoughts on instead using some verbal redirect, like the following:

```python
    meta: pd.DataFrame or pd.Series
        Empty pandas version of the result, see XXX for more information
```

I find myself somewhat irrationally allergic to adding decorators to methods for things like this.  There is some "technology used to benefit gained ratio" test that this fails for me.  It's hard for new developers to reason about what's going on without understanding all of the codebase.